### PR TITLE
[DEV-194] Deploy and verify

### PR DESCRIPTION
**Pull request by grp06**

### CODE REVIEW

**Comment by rahulrumalla** [comment]
Could you add some documentation in readme to clarify how/when to use this for developers, please?

**Review by rahulrumalla — CHANGES_REQUESTED** [review]

File: scripts/deploy_and_verify.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+source .env.rinkeby
```
[the remark this answers is not in the document]
Comment (grp06) [reply scripts/deploy_and_verify.sh]: done

**Comment by grp06** [comment]
> Could you add some documentation in readme to clarify how/when to use this for developers, please?



> Could you add some documentation in readme to clarify how/when to use this for developers, please?

done

File: readme.md [hunk 2]
```diff
@@ -67,6 +67,12 @@ npx hardhat typechain
 npx hardhat test
 ```
 
+### Deploying and verifying contract
+
+-   create a `.env` file or (`.env.whatever`) matching the parameters seen in `./scripts/deploy_and_verify.sh`
```
Comment (rahulrumalla) [inline readme.md:72]:
```suggestion
-   create a `.env` file or (`.env.network`) matching the parameters seen in `./scripts/deploy_and_verify.sh`, and pass it as a parameter when running the script as seen below
```

File: readme.md [hunk 3]
```diff
@@ -67,6 +67,12 @@ npx hardhat typechain
...
+-   run `./scripts/deploy_and_verify.sh .env.whatever`
```
Comment (rahulrumalla) [inline readme.md:73]:
```suggestion
-   run `./scripts/deploy_and_verify.sh .env.network`
```

**Comment by rahulrumalla** [comment]
Thanks for addressing previous comments. One addition comes to mind here i.e., providing a .env.example by convention that is not git ignored. This would help a developer know easily what keys they need rather than extrapolate it by understanding the script. Reduces human error this way I feel. Could we add .env.example and update readme docs as needed, please?

**Review by rahulrumalla — APPROVED** [review]

**Pull request merged**